enterprise_access/apps/api/v1/views/subsidy_access_policy.py

```diff
@@ -73,6 +75,60 @@
 GROUP_MEMBER_DATA_WITH_AGGREGATES_API_TAG = 'Group Member Data With Aggregates'
 
 
+def group_members_with_aggregates_next_page(current_url, page):
+    """Helper method to create the next page url"""
+    return current_url.replace(f"page={page}", f"page={int(page) + 1}")
+
+
+def group_members_with_aggregates_previous_page(current_url, page):
+    """Helper method to create the previous page url"""
+    return current_url.replace(f"page={page}", f"page={int(page) - 1}")
+
+
+def _format_group_members_with_aggregate_members_response(
```

Comment:
nit: maybe `def _pagination_params_for_group_aggregates(...)`

Reply:
as it's directly updating the values within member_response, I decided on `_update_pagination_params_for_group_aggregates`